bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/HSBType.java

```diff
@@ -227,27 +227,27 @@ public boolean equals(Object obj) {
         PercentType c = new PercentType(
                 value.multiply(BigDecimal.ONE.subtract((BigDecimal.ONE.subtract(f)).multiply(s))));
 
-        if (h_int == 0 || h_int == 6) {
+        if (hInt == 0 || hInt == 6) {
```

Comment:
Isn't that a candidate for a switch case (as you already touched it anyway)?

Reply:
You are absolutely right! I'll do it right away.